proposed/storageos-operator/v0.3.0/Chart.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: v1
-appVersion: "1.5.3"
+appVersion: "v2.1.0"
 description: Cloud Native storage for containers
 name: storageos-operator
-version: 0.2.19
+version: 0.3.0
 tillerVersion: ">=2.10.0"
 keywords:
 - storage
 - block-storage
 - volume
 - operator
 home: https://storageos.com
-icon: https://storageos.com/wp-content/themes/storageOS/images/logo.svg
+icon: https://docs.storageos.com/images/logo-helm.svg
```

Comment:
Please make a local copy of the logo in the parent folder `storageos-operator/` and reference the image here as `file://../logo-helm.svg`. This way the logo can be displayed in airtight environments.